common/navbar.js

```diff
@@ -388,7 +389,24 @@
                             rid: scope.ridSearchTerm
                         });
 
-                        $window.open(url, '_blank');
+                        // try to fetch the resolver link to see if the path resolves before sending the user
+                        // NOTE: .head returns a 405 error (not allowed) for proper RIDs, maybe ask Karl if this is expected
```

Comment:
If that's the case maybe we should just send a get request instead? I agree that we should ask Karl.

Reply:
From karl:
> I suspect 405 really means we didn't' implementn HEAD and it doesn't' mean anything